app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ApplicationPetsController < ApplicationController
+
+  def create
+  @application_pet = ApplicationPet.create(application_pet_params)
+
+    redirect_to show_application_path
+  end
+
+  def update
```

Comment:
Excellent adherence to RESTful conventions with these controller actions in the joins controller